src/app/(guest)/stats/[id]/page.tsx

```diff
@@ -31,32 +31,32 @@ const SubjectStats: React.FC<SubjectStatsProps> = async ({ params }) => {
   const subjectStats: ISubjectStats = await res.json();
 
   return (
-    <section className="w-full h-full flex flex-col items-center justify-center text-center overflow-x-hidden">
-      <p className="text-lg w-5/6 md:text-xl font-bold uppercase text-center px-4">
+    <section className="flex flex-col items-center justify-center w-full h-full overflow-x-hidden text-center">
+      <p className="w-5/6 px-4 text-lg font-bold text-center uppercase md:text-xl">
         As tuas <span className="text-primary">estatísticas </span>
         de <span className="text-primary">{subjectName}</span>
       </p>
-      <div className="max-w-7xl w-full flex flex-col py-6 px-8 gap-12">
-        <section className=" h-full flex flex-col gap-y-6">
-          <div className="flex items-center gap-x-2 text-xl bg-gray-100 px-6 py-4 rounded-md">
-            <div className="w-12 h-12 flex items-center justify-center">
+      <div className="flex flex-col w-full gap-12 px-8 py-6 max-w-7xl">
+        <section className="flex flex-col h-full gap-y-6">
+          <div className="flex items-center px-6 py-4 text-xl bg-gray-100 rounded-md gap-x-2 dark:bg-secondary-dark">
+            <div className="flex items-center justify-center w-12 h-12">
               <FiInfo className="w-full" />
             </div>
             <p className="text-xs md:text-lg">
-              Das <span className="text-primary font-bold">{subjectStats.total_of_questions}</span>{' '}
+              Das <span className="font-bold text-primary">{subjectStats.total_of_questions}</span>{' '}
               questões disponíveis respondeste a{' '}
-              <span className="text-primary font-bold">{subjectStats.n_of_answers}</span>, ou seja{' '}
-              <span className="text-primary font-bold">
+              <span className="font-bold text-primary">{subjectStats.n_of_answers}</span>, ou seja{' '}
+              <span className="font-bold text-primary">
                 {((subjectStats.n_of_answers / subjectStats.total_of_questions) * 100).toFixed(1)}%
               </span>
               .
             </p>
           </div>
         </section>
 
-        <div className="flex flex-col md:flex-row gap-y-8 md:gap-6 justify-between items-center px-6 py-4 bg-gray-100 rounded-md">
-          <div className="w-full flex flex-col justify-center items-center gap-2">
-            <div className="bg-primary text-white rounded-md w-full py-1">
+        <div className="flex flex-col items-center justify-between px-6 py-4 bg-gray-100 rounded-md md:flex-row gap-y-8 md:gap-6 dark:bg-secondary-dark">
+          <div className="flex flex-col items-center justify-center w-full gap-2">
+            <div className="w-full py-1 text-white rounded-md bg-primary">
               <p>Número de Exames</p>
             </div>
             <StatsPieChart
```

Comment:
In the charts you can use:

```
const options: ChartOptions<'CHART_TYPE'> = {
    color: 'white'
  };
```

In this case you would replace `CHART_TYPE` with the string `pie`

and then when rendering the `<Pie />` component:

`<Pie data={d} options={options} />`

Obviously you would need to change 'white' with a logic to check the active theme